client/src/App.js

```diff
@@ -1,31 +1,53 @@
-import { useEffect, useState } from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Users from './Components/Users';
+import { useState, useEffect } from 'react';
+// import './Components/form.css';
 
-function App() {
-  const [serverData, setServerData] = useState("");
 
-  useEffect(() => {
-    async function getServerData() {
-      const resp = await fetch('/api/hello');
-      const data = await resp.json();
 
-      console.log('Data from server:', data);
 
-      setServerData(data.message);
-    }
+function App() {
+  const [user, setUser] = useState('');
```

Comment:
`undefined` would probably be a better initial state, indicating it hasn't been fetched yet.